rpcs3/Emu/SysCalls/Modules/cellPngDec.cpp

```diff
@@ -3,572 +3,859 @@
 #include "Emu/System.h"
 #include "Emu/IdManager.h"
 #include "Emu/SysCalls/Modules.h"
-
-extern "C"
-{
-#include "stblib/stb_image.h"
-}
-
 #include "Emu/FS/VFS.h"
 #include "Emu/FS/vfsFileBase.h"
 #include "Emu/SysCalls/lv2/sys_fs.h"
 
+#include "png.h"
 #include "cellPngDec.h"
 
 extern Module<> cellPngDec;
 
-// cellPngDec aliases (only for cellPngDec.cpp)
-using PPMainHandle = vm::pptr<PngDecoder>;
-using PMainHandle = vm::ptr<PngDecoder>;
-using PThreadInParam = vm::cptr<CellPngDecThreadInParam>;
-using PThreadOutParam = vm::ptr<CellPngDecThreadOutParam>;
-using PExtThreadInParam = vm::cptr<CellPngDecExtThreadInParam>;
+// cellPngDec aliases to improve readability
```

Comment:
It's only like that, since they were like that earlier. The original goal was to make function headers shorter, and to improve reliability. Adding _ might invalidate the original goal.